repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/samlpolicy/SamlFlow10Test.groovy

```diff
@@ -113,19 +108,21 @@ class SamlFlow10Test extends ReposeValveTest {
         fakeIdentityV2Service.getIdpFromIssuerCount + fakeIdentityV2Service.getMappingPolicyForIdpCount == policyMappingQueries
 
         where:
-        isIt     | headerValue | samlIssuer                       | policyMappingQueries
-        "is"     | "1.0"       | "http://legacy.idp.external.com" | 0
-        "is not" | "2.0"       | generateUniqueIssuer()           | 2
+        contentType                 | isIt     | headerValue | samlIssuer                       | policyMappingQueries
+        APPLICATION_FORM_URLENCODED | "is"     | "1.0"       | "http://legacy.idp.external.com" | 0
```

Comment:
When I originally wrote this, I missed updating this string literal to using `SAML_LEGACY_ISSUER`.